### PR TITLE
Fix HTML validation errors

### PR DESCRIPTION
**Pull request by der-lyse**
This PR fixes several validation errors regarding invalid HTML.

I also found the cause of the [weird template syntax error I mentioned over there](https://github.com/jointwt/twtxt/issues/311#issuecomment-751458467). It was a Vim swap file, that was attempted to be parsed as a template, causing the panic at application startup. `git clean -x …` in the `clean` target of the _Makefile_ killed all of them, so things started to work again. I now exclude temporary editor files when loading the templates.

@prologic, since you're vision impaired, you may use screen readers or maybe can provide some insight on [alternative texts of images](https://github.com/jointwt/twtxt/commit/ba3808a0d60889e0bbc9ab18aa8b83bf6368f098). Not sure, whether there are blind users in the twtxt.net community who see this PR and can provide some feedback on the empty alternative text of user avatars.

### CODE REVIEW

**Comment by prologic** [comment]
> @prologic, since you're vision impaired, you may use screen readers or maybe can provide some insight on alternative texts of images. Not sure, whether there are blind users in the twtxt.net community who see this PR and can provide some feedback on the empty alternative text of user avatars.

Whilst I don't use a screen reader myself (_I use full screen zoom instead_), I _actually_ don't know how to handle this myself. I'm familiar with accessibility standards and I know what doesn't work for me, but I wouldn't have a clue what works and doesn't for a totally blind person.

That being said, _this_ is worth solving because it would open up decentralised social media (_without ads or tracking_) that's built on open web technologies to other vision impaired and blind users around the world (_there are some ~300M global vision impaired and blind persons in the world!_)

**Comment by prologic** [comment]
It's worth opening a separate issue to discuss what we can do here...

File: internal/templates/_partials.html [hunk 1]
```diff
@@ -94,16 +94,16 @@
       <div>
         {{ if $.User.Is $.Twt.Twter.URL }}
           <a href="{{ $.User.URL | trimSuffix "/twtxt.txt" }}" class="u-url">
-            <img class="avatar u-photo" src="/user/{{ $.User.Username }}/avatar" />
+            <img class="avatar u-photo" src="/user/{{ $.User.Username }}/avatar" alt="" />
```
Comment (prologic) [inline internal/templates/_partials.html:97]:
```suggestion
            <img class="avatar u-photo" src="/user/{{ $.User.Username }}/avatar" alt="Avatar of {{ $.User.Username }}" />
```

Perhaps this is good enough?

**Review by prologic — DISMISSED** [review]
LGTM 👍

**Review by prologic — APPROVED** [review]

**Comment by prologic** [comment]
I didn't notice this in the review sorry and I didn't test on mobile

Can you have a look at this regression?

image￼

**Comment by der-lyse** [comment]
> I didn't notice this in the review sorry and I didn't test on mobile

Damn, I didn't resize my browser small enough either. :-( I'll look into it!

**Pull request merged**